consumer.js

```diff
@@ -0,0 +1,14 @@
+"use strict"
```

Comment:
Let's put this logic into the main file. This module is responsible for pushing and pulling the queue so it's best to keep it all together. Lets change the pattern name to `role:updater,info:update` and update the main file calls as appropriate.